frontend/src/components/regions/CardDetailsModal/CardDetailsModal.test.js

```diff
@@ -0,0 +1,89 @@
+import { canSetDueTime } from "./utils";
+
+test("canSetDueTime function returns true if card belongs to current user and there's no due time set", () => {
+    const authUser = {
+        userId: 1,
+        permissions: {
+            teams: {
+                28: {
+                    id: 28,
+                    name: "Cohort 22 web dev",
+                    active: true,
+                    permissions: [
+                        "REVIEW_CARDS"
+                    ]
+                }
+            }
+        }
+    }
+    const card = {
+        assignees: [authUser.userId],
+        dueTime: null,
+    }
+    const teams = authUser.permissions.teams;
+    expect(canSetDueTime({card, authUser, teams})).toBe(true);
```

Comment:
Here you are effectively passing in teams twice. You are passing it in on the authUser object and as a seperate object. It is not useful to pass the same piece of information into the same function twice. What is this for?